src/client.js

```diff
@@ -70,6 +70,8 @@ function Client(manager, name, config = {}) {
 		attachedClients: {},
 		config: config,
 		id: uuidv4(),
+		idChan: 1,
+		idMsg: 1,
```

Comment:
Those are just for cosmetic purpose?

Reply:
Well message ids are used for loading more history and re connection logic.